debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/VirtualThreadsCommand.java

```diff
@@ -69,11 +69,16 @@ private static long getVmRef(J9ObjectPointer continuation) throws CorruptDataExc
 		return J9ObjectHelper.getLongField(continuation, vmRefOffset);
 	}
 
-	private static J9ObjectPointer getName(J9ObjectPointer vthread) throws CorruptDataException {
-		if (nameOffset == null) {
-			nameOffset = J9ObjectHelper.getFieldOffset(vthread, "name", "Ljava/lang/String;");
+	private static String getName(J9ObjectPointer vthread) throws CorruptDataException {
+		String name = "<N/A>";
+		if (vthread.notNull()) {
+			if (nameOffset == null) {
+				nameOffset = J9ObjectHelper.getFieldOffset(vthread, "name", "Ljava/lang/String;");
+			}
+			name = J9ObjectHelper.stringValue(J9ObjectHelper.getObjectField(vthread, nameOffset));
```

Comment:
This could use `J9ObjectHelper.getStringField()`:
```java
			name = J9ObjectHelper.getStringField(vthread, nameOffset);
```
Either way, `name` may be `null` and should be handled.

Reply:
fixed